File: PYTHON/sampah_elektronik.py
from sampah import Sampah

class SampahElektronik(Sampah):
    def __init__(self, id_sampah="", nama="", berat="", jenis_elektronik=""):
        super().__init__(id_sampah, nama, berat)
        self.__jenis_elektronik = jenis_elektronik

    # getter setter jenis elektronik
    def get_jenis_elektronik(self):
        return self.__jenis_elektronik

    def set_jenis_elektronik(self, jenis_elektronik):
        self.__jenis_elektronik = jenis_elektronik

File: CPP/sampahElektronik.cpp
#include <iostream>
#include <string>

using namespace std;

class SampahElektronik : public Sampah {
    // Atribut private
    private:
        string jenisElektronik; // Jenis sampah (Batre, Lampu, Dll).
    
    public:
        // constructor kosong
        SampahElektronik() {
        }

        // Constructor
        SampahElektronik(string idSampah, string nama, string berat, 
            string jenisElektronik) 
            : Sampah(idSampah, nama, berat) {
            this->jenisElektronik = jenisElektronik;
        }
        
        // getter setter jenis elektronik
        string getJenisElektronik() {return jenisElektronik;}

        void setJenisElektronik(string jenisElektronik) {this->jenisElektronik = jenisElektronik;}
        
    // Destructor
    ~SampahElektronik() {
    }
};